packages/python/src/components/CallSignature.tsx

```diff
@@ -1,24 +1,20 @@
 import {
-  Children,
   computed,
   createSymbolSlot,
   For,
   Show,
   SymbolSlot,
-  useContext,
 } from "@alloy-js/core";
 import { ParameterDescriptor } from "../parameter-descriptor.js";
 import { createPythonSymbol } from "../symbol-creation.js";
 import { PythonOutputSymbol } from "../symbols/index.js";
 import { Atom } from "./Atom.jsx";
-import { PythonSourceFileContext } from "./SourceFile.jsx";
+import { type TypeExpressionProps } from "./index.js";
```

Comment:
Best not to import from index. It will work but there will come a time when you are bit by circular reference problems and it will be awful. Ask me how I know 😂